src/types.ts

```diff
@@ -64,4 +64,11 @@ export type OrderDirection = 'ascending' | 'descending';
 export type Order<Field extends string | number | symbol> = {
   field: Field;
   direction: OrderDirection;
+
+  /**
+   * Indicates entries where the value for `field` is null/undefined should go
+   * last. Otherwise, they will get compared with non-null values normally.
```

Comment:
Ah. My intention was that nulls would always sort _either_ before or after non-null values. This would be `true` by default, or `false` to make them sort before other values.

Reply:
Done